src/Navbar.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const Navbar: React.FC = () => (
+  <nav className="navbar navbar-expand-lg bg-light justify-content-center">
+    <ul className="nav justify-content-center">
+      <li className="nav-item">
+        <Link to='/' className="nav-link">HOME</Link>
```

Comment:
```suggestion
        <Link to='/' className="nav-link">home</Link>
```
а апперкейс делай через css:
```
text-transform: 
```